src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -45,14 +45,24 @@ private static X509Certificate2Collection GetCertificateCollection(string certif
             {
                 // On Windows, .Net Core applications should not import PFX files in parallel to avoid a known system-level race condition.
                 // This bug results in corrupting the X509Certificate2 certificate state.
-                Assert.True(m.WaitOne(MutexTimeout), "Cannot acquire the global certificate mutex.");
-
-                var certCollection = new X509Certificate2Collection();
-                certCollection.Import(Path.Combine(TestDataFolder, certificateFileName), CertificatePassword, X509KeyStorageFlags.DefaultKeySet);
+                try
```

Comment:
> Or you're saying the only way to do it would be to store it to something in the store itself?

No, while you need to use PersistKeySet if you want to add it to a store, that flag just means "don't delete the keyfile on the SafeHandle release".  Opening the first instance with PersistKeySet and manually opening a CngKey for the private key and calling Delete on it at shutdown would work to allow safe clones.  But what's going to actually end up calling delete? There's not a Main() here to put a finally in.

And `orig` is... maybe usable in your sample.  If it's a public only, private via the store, private via EphemeralKeySet or private via PersistKeySet; then it works fine.  If it's private via perphemeral import then private key operations start failing, but public key operations still succeed.

(This message brought to you by: Sunburn.  Sunburn, the body's way of saying "you're a software developer, why did you go outside?" and "stay indoors")